app/vulncode_app/views.py

```diff
@@ -24,10 +24,45 @@
 import json
 
 from django.http import HttpResponse
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status
 
+from vulncode_app.serializers import ImpactedPackageSerializer
+from vulncode_app.models import ImpactedPackage
+from vulncode_app.models import Package
 from vulncode_app import api_data
 
 
+class VulnerabilityData(APIView):
+    def get(self, request, package_name):
+        vulnerability = []
+        extract = []
+        summary = []
+        reference_id = []
+        version = []
+
+        pk = Package.objects.filter(name=package_name)
+
+        for i, v in enumerate(pk):
```

Comment:
@kartiksibal rather than `for i, v in enumerate(pk):` just iterate on the packages:
```
packages = Package.objects.filter(name=package_name)
for package in packages:
  ....
```